inference/huggingface/test-gptj.py

```diff
@@ -24,13 +24,12 @@
 model = deepspeed.init_inference(model,
                                  mp_size=world_size,
                                  dtype=torch.float,
```

Comment:
let's change dtype to half here? Or we can get an input argument to set the data type.

Reply:
@RezaYazdaniAminabadi can you explain why we should change dtype to half here?

Reply:
since, our performance boosts with half precision. We also can get an input arg to switch between half and floating point.

Reply:
updated